packages/api/src/platforms/vtex/clients/search/index.ts

```diff
@@ -32,10 +32,11 @@ export interface ProductLocator {
 }
 
 export const IntelligentSearch = (
-  { account, environment, hideUnavailableItems }: Options,
+  { account, environment, hideUnavailableItems, workspace }: Options,
   ctx: Context
 ) => {
-  const base = `https://${account}.${environment}.com.br/api/io`
+  const accountBase = [workspace ?? 'master', account].join('--')
+  const base = `https://${accountBase}.${environment}.com.br/api/io`
```

Comment:
Don't need this, just pass the workspace as querystring. Also, IDK if we wanna add support for workspaces

Reply:
I think that add support to workspaces are a necessary in some cases to development phase. In my case, I'm developing the FastStore version for a store that is based on VTEX CMS, but has a VTEX IO enabled in determined workspace, without this support, the development is inviable.

How to pass the workspace as querystring?